playbooks/06-deploy-architecture.yml

```diff
@@ -128,6 +128,18 @@
                 msg: >-
                   Error detected. Check debugging output above.
 
+    - name: Set cifmw_architecture_automation_file if not set before
+      ansible.builtin.set_fact:
+        cifmw_architecture_automation_file: >-
+          {{
+            (
+              cifmw_architecture_repo | default(ansible_user_dir+'/src/github.com/openstack-k8s-operators/architecture'),
+              'automation/vars',
+              cifmw_architecture_scenario~'.yaml'
+            ) | ansible.builtin.path_join
+          }}
+      when: cifmw_architecture_automation_file is not defined
```

Comment:
usually the `when` is put after the `name` to make reading slightly easier

Reply:
ack, will change once the current CI run is finished